src/others/font_manager/lv_font_manager.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file lv_font_manager.h
+ *
+ */
+#ifndef LV_FONT_MANAGER_H
+#define LV_FONT_MANAGER_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../misc/lv_types.h"
+
+#if LV_USE_FONT_MANAGER
+
+#if !LV_USE_FREETYPE
+#error "LV_USE_FONT_MANAGER requires LV_USE_FREETYPE"
+#endif
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+typedef struct _lv_font_manager_t lv_font_manager_t;
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * Create main font manager.
+ * @param recycle_cache_size size of the recycle cache.
+ * @return pointer to main font manager.
+ */
+lv_font_manager_t * lv_font_manager_create(uint32_t recycle_cache_size);
+
+/**
+ * Delete main font manager.
+ * @param manager pointer to main font manager.
+ * @return return true if the deletion was successful.
+ */
+bool lv_font_manager_delete(lv_font_manager_t * manager);
+
+/**
+ * Add the font file path.
+ * @param manager pointer to main font manager.
+ * @param name font name.
+ * @param path font file path.
+ */
+void lv_font_manager_add_path(lv_font_manager_t * manager, const char * name, const char * path);
+
+/**
+ * Add the font file path with static memory.
+ * @param manager pointer to main font manager.
+ * @param name font name.
+ * @param path font file path.
+ */
+void lv_font_manager_add_path_static(lv_font_manager_t * manager, const char * name, const char * path);
```

Comment:
Name is usually short and I assume that there will be only a few font_manager. Is it really required to have a `*_static()` variant?

Reply:
The font file path is usually long, often reaching more than 100 characters.

Reply:
There are often more than 10 font files in a product, and using `static` makes sense.